pkgs/by-name/gr/gruvbox-kvantum/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  variant ? "Gruvbox-Dark-Brown",
+}:
+let
+  pname = "gruvbox-kvantum";
+in
+lib.checkListOfEnum "${pname}: variant"
+  [
+    "Gruvbox-Dark-Blue"
+    "Gruvbox-Dark-Brown"
+    "Gruvbox-Dark-Green"
+    "Gruvbox_Light_Blue"
+    "Gruvbox_Light_Brown"
+    "Gruvbox_Light_Green"
+  ]
+  [ variant ]
+
+  stdenvNoCC.mkDerivation
+  {
+    inherit pname;
+    version = "1.1";
+
+    src = fetchFromGitHub {
+      owner = "sachnr";
+      repo = "gruvbox-kvantum-themes";
+      rev = "f47670be407c1f07c64890ad53884ee9977a7db1";
+      sha256 = "sha256-u2J4Zf9HuMjNCt3qVpgEffkytl/t277FzOvWL8Nm8os=";
+    };
+
+    installPhase = ''
+      runHook preInstall
+      mkdir -p $out/share/Kvantum
+      cp -a "${variant}" $out/share/Kvantum
+      runHook postInstall
+    '';
+
+    meta = with lib; {
+      description = "Gruvbox themes for kvantum";
+      homepage = "https://github.com/sachnr/gruvbox-kvantum-themes";
+      license = licenses.gpl3;
+      platforms = platforms.linux;
+      maintainers = [ ];
```

Comment:
new packages must have maintainers. Add yourself to `maintainers/maintainer-list.nix` if you aren't one.

Reply:
I haven't seen this requirement

Reply:
I only saw

> We encourage people who care about a package to assign themselves as a maintainer

and I'm not sure, if I want to be the one, so I didn't add myself here

Reply:
https://github.com/NixOS/nixpkgs/tree/master/pkgs#quick-start-to-adding-a-package

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes